css/style.css

```diff
@@ -410,7 +411,7 @@ header {
   height: 24px;
   background: #ebebff;
   border-radius: 8px;
-  font-family: "Poppins", sans-serif;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] According to the requirements the Minimum mobile screen size should be 375px. however, between 375px and 490px the popup doesn't resemble the Microverse template anymore.
![afbeelding](https://user-images.githubusercontent.com/39240820/231572976-8567e052-7434-490e-8cae-4f01f46ff1e4.png)

- [ ] [OPTIONAL] In addition to that the rest of the application is also not responsive however, due to this not being a part of the current project milestones this will be optional.
![afbeelding](https://user-images.githubusercontent.com/39240820/231572506-c2b405d5-f4c8-4e8b-8008-ee601a8569d5.png)